core/templates/dev/head/services/local-storage.service.ts

```diff
@@ -61,7 +61,7 @@ export class LocalStorageService {
 
   /**
    * Check that localStorage is available to the client.
-   * @returns {boolean} true iff the client has access to localStorage.
+   * @returns {boolean} true if the client has access to localStorage.
```

Comment:
Why this change? I think iff here implies if & only if.

Reply:
I thought it was wrote wrong. Already fixed it.